app/_layout.tsx

```
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-require-imports */
-import { Appearance } from "react-native";
+import { Appearance, SafeAreaView } from "react-native";
```

ESLint

[error] 'SafeAreaView' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)